### DISCUSSION TITLE
דינמיקה תרגיל בית 10 שאלה 3

### DISCUSSION THREAD

**Discussion by WolfDan333** [NuclearGandhi/technion_second_brain#15] [טעויות ותיקונים]
בשאלה 3 מבקשים מאיתנו להגדיר מה הגודל והכיוון של כוח החיכוך. לפי פרופיל התנועה שיוצא בסוף עבור סעיפים ב ו-ג מתקבל גלגול שמאלה ואז גלגול ימינה, כלומר הכיוון של החיכוך משתנה.
לדעתך זה משפיע גם על הגודל שלו? ואולי גם על המשוואות הסופיות ככלל?

**Comment by NuclearGandhi** [comment]
לא כל כך הבנתי לאיזה שאלה אתה מתכוון, אין סעיפים ב' וג' בשאלה 3. בכל זאת, בכל זאת, גם ב-2 וגם ב-3 כיוון כוח החיכוך וכיוון הסיבוב באיור נבחרו שרירותית. בשום מקום לא דרשתי שהם יהיו חיוביים או שליליים, כך שזה לא אמור להשפיע על המשוואות או על הגודל שלו.

**Reply by WolfDan333** [reply]
קול תודה! והתבלבלתי באמת בין 2 ו3. הכוונה שלי הייתה לשאלה 2.
